language/move-lang/src/to_bytecode/context.rs

```diff
@@ -206,21 +204,21 @@ impl<'a> Context<'a> {
     //**********************************************************************************************
 
     fn ir_module_alias(ident: &ModuleIdent) -> IR::ModuleName {
-        let ModuleIdent_ { address, name } = &ident.0.value;
+        let (address, name) = &ident.value;
         IR::ModuleName::new(format!("{}::{}", address, name))
     }
 
     fn translate_module_ident(ident: ModuleIdent) -> IR::ModuleIdent {
-        let ModuleIdent_ { address, name } = ident.0.value;
-        let name = Self::translate_module_name(name);
+        let (address, name) = ident.value;
+        let name = Self::translate_module_name_(name);
```

Comment:
If `translate_module_name_` is only used here, would it be good to just inline it?

Reply:
Consistency mostly